common/Util.re

```diff
@@ -6,3 +6,12 @@ module ReactStuff = {
     external elementAsString: React.element => string = "%identity";
   };
 };
+
+module IntCmp =
```

Comment:
@baransu why was it necessary/useful to add IntCmp here?

Reply:
My little playground. Sorry 😛

Reply:
https://github.com/reason-association/reasonml.org/pull/35

Reply:
thx :)